documentation/src/main/markdown/runningdetect/basics/runningwithblackduck.md

```diff
@@ -27,13 +27,55 @@ Refer to [Black Duck Server properties](../../properties/configuration/blackduck
 ## Offline mode
 
 If you do not have a [blackduck_product_name] instance, or if your network is down, you can still run [solution_name] in offline mode.
+<note type="note">Offline mode is not the same as Air Gap mode. Air Gap mode requires the airgap.jar available to execute as it contains local copies of scanning libraries to support full offline execution.</note>
 In offline mode, [solution_name] writes output files (.bdio files and, when Vulnerability Impact Analysis runs, .bdmu files) to subdirectories
 within the run directory without attempting to upload them to [blackduck_product_name]. You can find the value of the run directory in the [solution_name] log.
 You can run [solution_name] in offline mode using the [offline mode property](../../properties/configuration/blackduck-server.md#offline-mode).
 
+### Running in offline mode
+
+Download the latest [solution_name] version:
+ [See download locations](https://sig-product-docs.synopsys.com/bundle/integrations-detect/page/downloadingandinstalling/downloadlocations.html)   
+ 
+*Choose one of the following to download.*
+
+**Version 8.7 and later**   
+`synopsys-detect-X.X.X-air-gap-no-docker.zip` includes scanning for Gradle and Nuget no Docker
+`synopsys-detect-X.X.X-air-gap.zip` includes scanning for Gradle, Nuget, or Docker
+
+Download the Signature Scanner from your [blackduck_product_name] server:
+https://{blackduckserver}/download/scan.cli.zip
+https://{blackduckserver}/download/scan.cli-windows.zip
+https://{blackduckserver}/download/scan.cli-macosx.zip
+
+Scanning [solution_name] Properties to specify:
+* --blackduck.offline.mode=true
+* --detect.scan.output.path= output of the signature scanner
+* --detect.output.path= output of the dependency scanner
```

Comment:
What is "dependency scanner"? I'm assuming this is Detect.

Reply:
Ya, let me reword,